src/tools/ceph_objectstore_tool.cc

```diff
@@ -2399,6 +2400,22 @@ int print_obj_info(ObjectStore *store, coll_t coll, ghobject_t &ghobj, Formatter
       formatter->close_section();
     }
   }
+  bufferlist hattr;
+  gr = store->getattr(ch, ghobj, ECUtil::get_hinfo_key(), hattr);
```

Comment:
i think collection handle is a relatively new thing -- it was introduced in mimic. probably you could take a look 38d24fefd6fa80c9f85afc9e0e4570b6fe387a7e for some inspirations?

Reply:
@tchaikov yes, thank you for the pointers. I have replaced the 'ch' to 'coll' as per previous calls to 'store->getattr' and also changed the 'cbegin()' to 'begin()'.